src/main/scala/extracells/container/ContainerFluidStorage.java

```diff
@@ -260,21 +266,48 @@ public boolean fillSecondSlot(ItemStack itemStack) {
 					return false;
 				}
 				this.storageCell.usePower(this.player, 20.0D,
-						this.player.getCurrentEquippedItem());
+					this.player.getCurrentEquippedItem());
 			}
 			this.inventory.incrStackSize(1, itemStack.stackSize);
 			return true;
 		}
 	}
-
 	public void forceFluidUpdate() {
-		if (this.monitor != null)
+		if (this.monitor != null) {
+			new PacketFluidStorage(this.player, this.monitor.getStorageList())
+				.sendPacketToPlayer(this.player);
+			new PacketFluidStorage(this.player, this.hasWirelessTermHandler)
+				.sendPacketToPlayer(this.player);
+		}
+	}
+
+	public void FluidUpdateOnce() {
+		if (this.monitor != null && this.UpdateOnce) {
 			new PacketFluidStorage(this.player, this.monitor.getStorageList())
-					.sendPacketToPlayer(this.player);
-		new PacketFluidStorage(this.player, this.hasWirelessTermHandler)
 				.sendPacketToPlayer(this.player);
+			new PacketFluidStorage(this.player, this.hasWirelessTermHandler)
+				.sendPacketToPlayer(this.player);
+			this.UpdateOnce = false;
+		}
 	}
 
+	public void forceFluidUpdate(String text) {
```

Comment:
```suggestion
	public void forceFluidUpdate(String searchText) {
```
To make it clear what `text` refers to